src/main/java/seedu/manager/command/AddCommand.java

```diff
@@ -1,20 +1,29 @@
 package seedu.manager.command;
 
+import seedu.manager.event.EventList;
+
 /**
  * Represents an executable add command
  */
 public class AddCommand extends Command {
     public static final String COMMAND_WORD = "add";
-    private static final String ADD_MESSAGE = "The following event has been added to the event list: %1$s";
+    public static final String ADD_MESSAGE = "Event added successfully";
```

Comment:
I think that since this constant is only used within the class, could it be made private?